.laminas-ci.json

```diff
@@ -1,5 +1,6 @@
 {
     "ignore_php_platform_requirements": {
-        "8.3": true
+        "8.3": true,
```

Comment:
Version 8.3 is still needed here?

Reply:
Yes: https://github.com/laminas/laminas-servicemanager/actions/runs/11406081441